src/main/kotlin/racingcar/domain/car/vo/Name.kt

```diff
@@ -0,0 +1,10 @@
+package racingcar.domain.car.vo
+
+private const val MAXIMUM_NAME_LENGTH = 5
+
+class Name(val value: String) {
```

Comment:
```suggestion
@JvmInline
value class Name(val value: String) {
```

value class도 사용이 가능해 보이네요! value class를 사용하면 장단점이 무엇이 있을까요? 😁

참고
- [inline classes](https://kotlinlang.org/docs/inline-classes.html)